phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/PhoenixScrutinyJobCounters.java

```diff
@@ -41,5 +41,11 @@
     /**
      * Number of batches processed
      */
-    BATCHES_PROCESSED_COUNT;
+    BATCHES_PROCESSED_COUNT,
+    /**
+     * Number of rows in source that became older than the max lookback age while scrutiny
+     * was comparing them with the target, and didn't match. We break these out separately because
+     * they could be due to extra versions being compacted, and are harmless.
+     */
+    BEYOND_MAX_LOOKBACK;
```

Comment:
All counters end with '_COUNT' here ... Perhaps we should remain consistent. That said, 'BEYOND_MAX_LOOKBACK_ROW_COUNT' looks too long. How about 'NO_LOOKBACK_ROW_COUNT'?

Reply:
No objection to appending _COUNT to the existing enum value, but NO_LOOKBACK_ROW_COUNT seems more confusing to me.

Reply:
True. They are similar to 'EXPIRED' but can't use the same name... 'UNREADABLE' can also be confusing. Anyways, pick what sounds best to you.